src/app/ui/components/Account/SendOnActivity/SendOnActivity.py

```
@@ -0,0 +1,197 @@
+from src.app.ui.components.Account.CompactAccountList.CompactAccountList import CompactAccountList
+from src.app.ui.components.Account.SendOnActivity.DateSelector.DateSelector import DateSelector
+from src.app.ui.components.Account.SendOnActivity.DestinationSelector.DestinationSelector import DestinationSelector
+from src.app.ui.components.Account.SendOnActivity.DestinationSelector.utils.DistanceCounter import DistanceCounter
+from src.app.ui.components.Account.SendOnActivity.utils.ActivityTypeSelector import ActivityTypeSelector
+from src.database.DbHandler import DbHandler
+
+
+class SendOnActivity(QWidget):
+    destinationChanged = pyqtSignal(dict)
+
+    def __init__(self, account_id=None, activity_type=None, start_point=None, end_point=None, parent=None, selected_accounts=[]):
```

Ruff (0.11.9)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)